projects/objects/lights/protos/CeilingLight.proto

```diff
@@ -12,7 +12,9 @@ PROTO CeilingLight [
   field SFColor    supportColor               1 1 1                                    # Defines the color of the light support.
   field MFString   supportTextureUrl          "textures/light_support_base_color.jpg"  # Defines the texture used for the light support.
   field SFColor    pointLightColor            1 1 1                                    # Defines the color of the point light.
+  field SFVec3f    pointLightAttenuation      0 0 1                                    # Defines the attenuation of the point light.
```

Comment:
I recently dropped the attenuation of every lights. Normally, an equivalent result is achievable using a `0 0 1` attenuation and playing with the `intensity` field only.